lib/activerecord-clean-db-structure/clean_dump.rb

```diff
@@ -151,7 +151,7 @@ def order_column_definitions(source)
           inside_table = true
           columns = []
           result << source_line
-        elsif source_line.start_with?(");")
+        elsif source_line.start_with?(")")
```

Comment:
Test fails without this fix:

![image](https://github.com/user-attachments/assets/d1ffeba1-c5e4-48bf-87df-6b5978dac866)